app/common/auth/authentication.service.js

```diff
@@ -22,11 +22,21 @@ function (
     ModalService
 ) {
 
-    // check whether we have initially an valid access_token and userId
-    // and assume that, if yes, we are still loggedin
-    var loginStatus = !!Session.getSessionDataEntry('accessToken') &&
-        Session.getSessionDataEntry('accessTokenExpires') > Math.floor(Date.now() / 1000) &&
-        !!Session.getSessionDataEntry('userId');
+    // check whether we have initially an valid access_token and assume that, if yes, we are still loggedin
```

Comment:
@crcommons I restructured this a bit to make Auth service responsible for clearing auth state if the token has expired.